docs/Downloading-Controlled-Access-Data.md

```diff
@@ -0,0 +1,48 @@
+A user authentication token (data access token) is required for downloading controlled-access data programmatically from the cBioPortal. See [Obtaining an Authentication Token](#obtaining-an-authentication-token) for instructions on how to download a token and [Using Authentication Tokens](#using-authentication-tokens) for how to use it.
+
+## [Obtaining an Authentication Token](obtaining-an-authentication-token)
+To obtain an authentication token, simply select the `Download Token` option from the dropdown menu at the top right corner of the cBioPortal home page (where your username is displayed). Your token will be downloaded as file with the following format:
+
+```
+token: your-authentication-token
+creation_date: YYYY-MM-DD HH:MM:SS
+expiration_date: YYYY-MM-DD HH:MM:SS
+```
+
+_Note that the expiration date may vary between different controlled-access instances of the cBioPortal._
+
+## [Using Authentication Tokens](using-authentication-tokens)
+Once downloaded, the token can be sent in the `Authorization` header when making requests to the [cBioPortal web API](cBioPortal-Web-API.md).
+```
+Authorization: Bearer <token>
+```
+
+## [Implementing Controlled-Access Tokens](implementing-controlled-access-tokens)
+To implement controlled-access tokens for your cBioPortal instance:
+
+1. Decide on an implementation ([JWT](#jwt-implementation) or [UUID](#uuid-implementation)). 
+2. Add or modify your `portal.properties` with the following:
+
+
+|           Property          |                                      Description                                          |    Default    |      Type Allowed     |
+| --------------------------- | ----------------------------------------------------------------------------------------- | ------------- | --------------------- |
+| dat.unauth_users            | Users that should never be allowed to download an authentication token                    | anonymousUser | comma-delmited string |
+| dat.method                  | Authentication token implementation method                                                | none          | jwt, uuid, none       |
+| dat.ttl_seconds             | Time in seconds until token expiration                                                    | -1            | integer               |
+| jwt.secret_key ($)          | Signature used to verify the token is valid and not modified.                             | none          | string                |
+| jwt.issuer ($)              | Issuer of the token.                                                                      | none          | string                |
+| dat.revoke_other_tokens (*) | Determines whether existing tokens should be revoked if a new token is created for a user | true          | true or false         |
+| dat.max_number_per_user (*) | Maximum number of tokens allowed per user                                                 | -1            | integer               |
+
+- \* UUID-specific implementation
```

Comment:
I would say "Options with this footnote are only applicable when using the UUID token format"